test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -326,6 +326,32 @@ spec setup =
             actual = t.fill_nothing ["col0", "col1"] default
             actual.at "col0" . to_vector . should_equal [0, 1000, 4, 5, 1000, 1000]
             actual.at "col1" . to_vector . should_equal [1000, 200, 1000, 400, 500, 1000]
+        
+        Test.specify "should allow to fill_nothing from larger string and expands type to fit" <|
+            t = table_builder [["col0", ["0", Nothing, "4", "5", Nothing, Nothing]]] . cast "col0" (Value_Type.Char size=1 variable_length=False)
+            actual = t.fill_nothing ["col0"] "ABCDE"
+            actual.at "col0" . to_vector . should_equal ["0", "ABCDE", "4", "5", "ABCDE", "ABCDE"]
+            if setup.is_database.not then
+                actual.at "col0" . value_type . should_equal (Value_Type.Char size=5 variable_length=True)
```

Comment:
Do you think we could check this invariant for Postgres?

Reply:
I wasn't sure how to specify "don't test this on SQLite". Do we have anywhere else we do this?

Reply:
1. One way is to check the prefix which contains the name of the DB: https://github.com/enso-org/enso/blob/develop/test/Table_Tests/src/Common_Table_Operations/Filter_Spec.enso#L85 But I think that's not quite perfect.

2. The preferred way is to use 'feature flags' from test selection, and I think `setup.test_selection.fixed_length_text_columns` is exactly what is needed here: 
https://github.com/enso-org/enso/blob/develop/test/Table_Tests/src/Common_Table_Operations/Conversion_Spec.enso#L76

Reply:
Thank you